linux_os/guide/system/bootloader-grub2/non-uefi/grub2_password/rule.yml

```diff
@@ -27,10 +27,12 @@ description: |-
     root account and password.
     {{% endif %}}
     <br /><br />
+    {{% if product in ["sle12", "sle15"] %}}
     Once the superuser password has been added,
     update the
     <tt>grub.cfg</tt> file by running:
     <pre>grub2-mkconfig -o {{{ grub2_boot_path }}}/grub.cfg</pre>
+    {{% endif %}}
```

Comment:
This section is missing the `update-grub` command for `ubuntu`.
I guess you can copy it from the uefi rule.